src/App.js

```diff
@@ -23,6 +23,8 @@ function App() {
   const [postBody, setPostBody] = useState("")
   const [openModal, setOpenModal] = useState(false)
   const [loginStatus, setLoginStatus] = useState("")
```

Comment:
We could use login status as boolean to False at first, and set true if response is error.  Cleaner code :-)

Reply:
loginStatus should be string because then the string will be displayed. There are two type of errors (from what I saw in backend) so the state will change according to which error occur at the time.